policyengine_us/variables/gov/states/ms/tax/income/ms_agi_subtractions.py

```diff
@@ -0,0 +1,16 @@
+from policyengine_us.model_api import *
+
+
+class ms_agi_subtractions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Mississippi adjustments to federal adjusted gross income"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://www.dor.ms.gov/sites/default/files/Forms/Individual/80100221.pdf#page=13",
```

Comment:
I think if Mississippi calls them adjustments - we should rename everything to adjustments from subtractions